src/ml/neural_net/tf_compute_context.cpp

```diff
@@ -317,14 +318,22 @@ std::unique_ptr<model_backend> tf_compute_context::create_activity_classifier(
   const float* pred_window = prediction_window.data();
   int pw = static_cast<int>(*pred_window);
 
+  int seed = FLEX_UNDEFINED;
+  auto random_seed_it = config.find("random_seed");
+  if (random_seed_it != config.end()) {
```

Comment:
what happens if FLEX_UNDEFINED is passed into the tensorflow code?
I guess in the inference time it doesn't matter but just want to make sure.

Reply:
FLEX_UNDEFINED is like None in python side. Also it doesn't matter because seed is used in dropouts which is disabled at inference time.